recipes/dpu-utils/meta.yaml

```
@@ -0,0 +1,62 @@
+{% set name = "dpu-utils" %}
+{% set version = "0.4.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/dpu_utils-{{ version }}.tar.gz
+  sha256: d385083cb6dde43d9f469dc8627dfd13f25d95789890b51b55f90ebeefa15311
+
+build:
+  number: 0
+  skip: True  # [not linux]
+  skip: True  # [py>37]
```

Looks like the package could be `noarch`
```suggestion
    noarch: python
```

---

Unfortunately, the `noarch` options doesn't work: I have tested in https://github.com/conda-forge/staged-recipes/pull/15363/commits/f7e88c59435048e69e120769ec542a0b79452028.

---

There is no build log anymore. What was the issue?

---

Even I don't remember, let's see what https://github.com/conda-forge/staged-recipes/pull/15363/commits/ab09e11bf492a222ad6545f010ac0e34f5596f5f generates. Resolving this conversation.